docs/api/apiv3/components/schemas/work_packages_model.yml

```diff
@@ -1,6 +1,43 @@
 # Schema: Work_PackagesModel
 ---
 type: object
+required:
+  - _embedded
+  - _type
+properties:
+  _type:
+    type: string
+    enum:
+      - Collection
+  count:
+    type: integer
+    minimum: 0
+  total:
+    type: integer
+    minimum: 0
+  _links:
+    type: object
+    required:
+      - self
+    properties:
+      self:
+        allOf:
+          - "$ref": "./link.yml"
+          - description: |-
+              This collection
+  
+              **Resource**: N/A
+            readOnly: true
+  _embedded:
+    type: object
+    required:
+      - elements
+    properties:
+      elements:
+        type: array
+        items:
+          $ref: '#/components/schemas/Work_PackageModel'
```

Comment:
A bit unrelated: I just came across this line and found `Work_PackageModel` to be very ugly. Is there a need of mixing camel case and snake case? Here Work and PackageModel look like two different words. Can we drop upcase in file names?